VimeoNetworking/Sources/VimeoSessionManager+Constructors.swift

```diff
@@ -132,17 +132,6 @@ public extension VimeoSessionManager
     
     private static func backgroundSessionConfiguration(identifier identifier: String) -> NSURLSessionConfiguration
     {
-        let sessionConfiguration: NSURLSessionConfiguration
-        
-        if #available(iOS 8.0, OSX 10.10, *)
-        {
-            sessionConfiguration = NSURLSessionConfiguration.backgroundSessionConfigurationWithIdentifier(identifier)
-        }
-        else
-        {
-            sessionConfiguration = NSURLSessionConfiguration.backgroundSessionConfiguration(identifier)
-        }
-        
-        return sessionConfiguration
+        return NSURLSessionConfiguration.backgroundSessionConfigurationWithIdentifier(identifier)
```

Comment:
Yeah, the deployment target for the pod is set for iOS 8+ and tvOS 9+. The deployment target for the example project is 9.3, but we could change that if we need to.